.travis.yml

```diff
@@ -12,7 +12,7 @@ install:
 script:
     - tox -e $TOX_ENV
     - ./node_modules/.bin/grunt --verbose
-    - flake8 kickoff/ --exclude=kickoff/test/ --max-line-length=2000 --ignore=E402
+    - flake8 kickoff/ --exclude=kickoff/test/ --max-line-length=2000 --ignore=E402,E711
```

Comment:
You don't need ```filter(something != None)``` see http://docs.sqlalchemy.org/en/latest/orm/tutorial.html#common-filter-operators

Instead you can use ```filter(something.isnot(None))```